pkg/network/config/config.go

```diff
@@ -156,7 +156,7 @@ func NewDefaultConfig() *Config {
 		MaxClosedConnectionsBuffered: 50000,
 		MaxConnectionsStateBuffered:  75000,
 		MaxDNSStatsBufferred:         75000,
-		MaxHTTPStatsBuffered:         75000,
+		MaxHTTPStatsBuffered:         100000,
```

Comment:
Why this change?

Reply:
I was trying to get to a number where 1) we wouldn't drop stats for a reasonable request rate 2) result in an acceptable memory usage

In my tests this number is able to process ~3500 requests/s staying around 350MiB